src/legacy/core_plugins/data/public/index.ts

```diff
@@ -17,33 +17,33 @@
  * under the License.
  */
 
-import { SearchBarService } from './search_bar';
-import { QueryBarService } from './query_bar';
+import { SearchService } from './search';
+import { QueryService } from './query';
 import { IndexPatternsService, IndexPatternsSetup } from './index_patterns';
 
 class DataPlugin {
   private readonly indexPatterns: IndexPatternsService;
-  private readonly searchBar: SearchBarService;
-  private readonly queryBar: QueryBarService;
+  private readonly search: SearchService;
+  private readonly query: QueryService;
 
   constructor() {
     this.indexPatterns = new IndexPatternsService();
-    this.queryBar = new QueryBarService();
-    this.searchBar = new SearchBarService();
+    this.query = new QueryService();
+    this.search = new SearchService();
   }
 
   public setup() {
     return {
       indexPatterns: this.indexPatterns.setup(),
-      search: this.searchBar.setup(),
-      query: this.queryBar.setup(),
+      search: this.search.setup(),
+      query: this.query.setup(),
```

Comment:
The `SearchSetup` and `QuerySetup` types also need to be added to the `DataSetup` interface below. (And we also need to re-export any other public types below too)

Reply:
Fixed

Not sure still which types I'm going to export.
WIll add as needed.